kpm/README.adoc

```diff
@@ -26,7 +26,9 @@ Note that this installation method assumes `/bin/bash` to be available on your s
 === Through Rubygems
 image:https://img.shields.io/gem/v/kpm?color=blue&label=kpm[kpm]
 
-On Windows, you can install KPM via https://rubygems.org/gems/kpm[RubyGems.org]. For this, you need to first install Ruby. Ruby can be installed via https://rubyinstaller.org/[RubyInstaller]. Once installed, you can run the following command to install KPM:
+On Windows, you can install KPM via https://rubygems.org/gems/kpm[RubyGems.org]. For this, you need to first install Ruby 2.x.x. We recommend installing `Ruby 2.7.7`.
```

Comment:
@vnandwana - You can remove the code syntax around Ruby 2.7.7.

Reply:
Consider rephrasing to `The recommended Ruby version is 2.7.7 at the time of writing`.

Reply:
Maybe explicitly mention that kpm does not work with Ruby 3.x? I'll let you decide. You can add it as a separate note if at all you decide to add it.